src/gui/mainwindow.cpp

```diff
@@ -1192,7 +1192,18 @@ void MainWindow::closeEvent(QCloseEvent *e)
     const bool goToSystrayOnExit = pref->closeToTray();
     if (!m_forceExit && m_systrayIcon && goToSystrayOnExit && !this->isHidden())
     {
-        e->ignore();
+        // This is the same as QX11Info::isPlatformX11(), inlined manually to avoid Qt X11 Extras modules
```

Comment:
I would omit this comment since `QX11Info` is obsoleted.
https://doc.qt.io/qt-5/qx11info.html